setup.py

```diff
@@ -3,19 +3,21 @@
 import glob
 import os
 
-__version = "11.2.0.1"
+__version = "1.0.0"
```

Comment:
Bad idea.
1. We have children who specified this version as minimal supported. They will be broken.
2. Django migrations may fail because applications version may be stored here.
Please restore original version and increase it.